src/apis/projects/api.ts

```diff
@@ -0,0 +1,39 @@
+import axios from 'axios';
+
+import {PaginatedResponse} from 'types/api';
+import {Project, Milestone} from 'types/projects';
+
+export async function getProjects(): Promise<Project[]> {
+  const response = await axios.get<PaginatedResponse<Project>>(`${process.env.REACT_APP_BACKEND_API}/projects`);
+
+  if (!response.data) {
+    throw new Error('Error while fetching projects, please try again.');
+  }
+  return response.data.results;
+}
+
+export async function getMilestones(): Promise<Milestone[]> {
+  const response = await axios.get<PaginatedResponse<Milestone>>(`${process.env.REACT_APP_BACKEND_API}/milestones`);
+  if (!response.data) {
+    throw new Error('Error while fetching milestones, please try again.');
+  }
+  return response.data.results;
+}
+
+export async function getMilestonesByProject(id: string): Promise<Milestone[]> {
+  const response = await axios.get<PaginatedResponse<Milestone>>(
+    `${process.env.REACT_APP_BACKEND_API}/milestones?project=${id}`,
```

Comment:
@angle943 just a heads up, it seems like the filter for query `project` is not up yet, so all milestones will be returned instead of the specific project's.

Reply:
Removed the milestone related apis as the `milestones` is now nested within the `projects` model